config/clients/java/template/Java/libraries/native/api_doc.mustache

```diff
@@ -0,0 +1,280 @@
+# {{classname}}{{#description}}
```

Comment:
same, delete

Reply:
Deleted in [3dbbaf7](https://github.com/openfga/sdk-generator/pull/158/commits/3dbbaf752f74a14f3535e48df92b4a934822699d)